domains/pallets/auto-id/src/lib.rs

```diff
@@ -360,11 +376,6 @@ impl<T: Config> Pallet<T> {
                         Error::<T>::ExpiredCertificate
                     );
 
-                    ensure!(
```

Comment:
Shouldn't we still check if the issuer cert is revoked before issuing a leaf cert ?